README.md

```diff
@@ -312,4 +312,4 @@ deprecate it for 18 months before removing it, if possible.
 [stackdriver-sample]: https://github.com/census-instrumentation/opencensus-csharp/blob/develop/src/Samples/TestStackdriver.cs

```

Comment:
should it all point to open-telemetry/opentelemetry-dotnet instead of census?

Reply:
yeah. this PR was just to test CI